service/src/main/java/bio/terra/cbas/runsets/monitoring/SmartRunsPoller.java

```diff
@@ -72,7 +72,8 @@ public void updateOutputAttributes(Run run)
     List<WorkflowOutputDefinition> outputDefinitionList =
         objectMapper.readValue(run.runSet().outputDefinition(), new TypeReference<>() {});
     Object outputs = cromwellService.getOutputs(run.engineId());
-    RecordAttributes outputParamDef = OutputGenerator.buildOutputs(outputDefinitionList, outputs);
+    Map<String, Object> outputParamDef =
+        OutputGenerator.buildOutputs(outputDefinitionList, outputs);
```

Comment:
`RecordAttributes` is gone, replaced by `Map<String, Object>` as a result of changing from swagger-codegen to openapi-generator. I am still investigating if this can be reverted.

Reply:
this is (hopefully) no longer true; resolving